includes/PF_CreatePageJob.php

```diff
@@ -70,7 +70,7 @@ public static function createOrModifyPage( $wikiPage, $pageText, $editSummary, $
 		}
 
 		$updater = $wikiPage->newPageUpdater( $user );
-		$updater->setContent( MediaWiki\Storage\SlotRecord::MAIN, $newContent );
+		$updater->setContent( MediaWiki\Revision\SlotRecord::MAIN, $newContent );
```

Comment:
please don't mix fixes with enhancements in one PR, otherwise tracking the changelog and semantic versioning becomes unnecessary hard

Reply:
In this case I think the fix was OK because the code simply didn't work

Reply:
@gesinn-it-gea Alexander, can this be merged or do you want more improvements ?